spec/api_v1.yaml

```diff
@@ -0,0 +1,368 @@
+openapi: "3.0.2"
+info:
+  description: >
+    Cachito is a service that stores application sources and their dependencies.
+  license:
+    name: "GPL-3.0"
+  title: Cachito API
+  version: "1.0"
+servers:
+  - url: https://cachito.domain.local/api/v1
+paths:
+  /requests:
+    get:
+      description: Returns all the Cachito requests
```

Comment:
The wording `Returns all the Cachito requests` is in a different verb tense than `Create a new request`. FWIW, I think "Return all the Cachito requests" would look better in the UI context.